package.json

```diff
@@ -20,12 +20,14 @@
 		"find-java-home": "^1.1.0",
 		"globby": "^11.0.0",
 		"html-escaper": "^3.0.0",
+		"is-zip": "^1.0.0",
+		"istextorbinary": "^5.12.0",
 		"mustache": "^4.0.1",
 		"node-stream-zip": "1.11.3",
 		"normalize-path": "^3.0.0",
 		"picomatch": "^2.2.2",
 		"reflect-metadata": "^0.1.13",
-		"retire": "^2.2.3",
+		"retire": "^2.2.4",
```

Comment:
Do we need to update the versions.yml for this retire upgrade?

Reply:
Good catch. I'll get a PR ready shortly.